test/src/test/java/org/corfudb/infrastructure/logreplication/SessionManagerTest.java

```diff
@@ -52,18 +57,21 @@ public void testSessionMgrWithOutgoingSession() {
      */
     @Test
     public void testSessionMgrWithIncomingSession() {
-        boolean sinkClusterAsLocalEndpoint = true;
-        topology = new DefaultClusterManager(sinkClusterAsLocalEndpoint).generateDefaultValidConfig();
+        DefaultClusterManager defaultClusterManager = new DefaultClusterManager();
+        DefaultClusterConfig topologyConfig = new DefaultClusterConfig();
+        defaultClusterManager.setLocalNodeId(topologyConfig.getSinkNodeUuids().get(0));
+        topology = defaultClusterManager.generateDefaultValidConfig();
+        
         SessionManager sessionManager = new SessionManager(topology, corfuRuntime);
         String sinkClusterId = "456e4567-e89b-12d3-a456-556642440002";
-        int numSinkCluster = topology.getSinkClusters().size();
+        int numSourceCluster = topology.getRemoteSourceClusters().size();
 
         // Verifies that the sink cluster has established session with all 3 source clusters.
```

Comment:
Here, should it validate 1 or 3 source clusters?

Reply:
its 1 now. Will correct the comment in the next push